src/main/java/org/terasology/polyworld/IslandWorldGenerator.java

```diff
@@ -102,11 +102,11 @@ public Vector3f getSpawnPosition(EntityRef entity) {
         GraphFacet graphs = worldRegion.getFacet(GraphFacet.class);
         WhittakerBiomeModelFacet model = worldRegion.getFacet(WhittakerBiomeModelFacet.class);
         Vector2f pos2d = new Vector2f(pos.getX(), pos.getZ());
-        CirclePickerClosest<org.terasology.polyworld.graph.Region> picker = new CirclePickerClosest<>(pos2d);
+        CirclePickerClosest<org.terasology.polyworld.graph.GraphRegion> picker = new CirclePickerClosest<>(pos2d);
```

Comment:
This should now be possible without the fully qualified name (as there is no name clash anymore)
```suggestion
        CirclePickerClosest<GraphRegion> picker = new CirclePickerClosest<>(pos2d);
```
This change would likely need an import, though.